lib/rules/selector-max-id/__tests__/index.mjs

```diff
@@ -524,3 +541,88 @@ testRule({
 		},
 	],
 });
+
+// with includeContextFunctionalPseudoClasses optional params
+
+testRule({
+	ruleName,
+	config: [
+		0,
+		{
+			includeContextFunctionalPseudoClasses: [':--foo'],
+		},
+	],
+
+	accept: [
+		{
+			code: ':--foo { --fizz: 1px;}',
```

Comment:
```suggestion
			code: ':--bar() {}',
```

To follow [conventions](https://github.com/stylelint/stylelint/blob/15.11.0/docs/developer-guide/rules.md#write-tests).